### DISCUSSION TITLE
Add `on_air` functionality to `ui.run_with()`

### DISCUSSION THREAD

**Discussion by csrubin** [zauberzeug/nicegui#2526] [Ideas / Feature Requests]
Using `on_air` to quickly test and share applications is a _great_ feature! However, it is only available with the standard `ui.run()` function. We are using the `ui.run_with()` function to more explicitly develop a FastAPI app with nicegui as the frontend, but this does not support `on_air` which makes sharing/testing more difficult for a distributed team. 

We are not ready to fully deploy/host the application yet, so we're stuck with pushing/pulling local branches for testing. Not the end of the world, but I'd love to see this functionality added!

**Comment by falkoschindler** [comment]
Thanks for the suggestion, @csrubin!
I just created PR #2546 to implement this feature. 🙂

**Reply by csrubin** [reply]
Wow so fast! I actually tried the exact same thing but apparently I tested it wrong. Next time I'll try to PR changes if I think I understand the issue

**Comment by falkoschindler** [comment]
This feature has just been released in version [1.4.16](https://github.com/zauberzeug/nicegui/releases/tag/v1.4.16). 🎉
